From c2cb0991fd03a5eeceb84edb797d035de8009281 Mon Sep 17 00:00:00 2001
From: Tim Stahlhut <stahta01@gmail.com>
Date: Sun, 6 Dec 2020 01:33:20 -0500
Subject: msys remove g_option_context_set_delocalize

glib goption.c has been patched to no longer need this fix.

---
 src/main-application.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/main-application.c b/src/main-application.c
index ae8beaccd..a00eed642 100644
--- a/src/main-application.c
+++ b/src/main-application.c
@@ -111,12 +111,6 @@ gnumeric_arg_parse (int argc, char **argv)
 	g_option_context_add_main_entries (ocontext, gnumeric_options, GETTEXT_PACKAGE);
 	g_option_context_add_group	  (ocontext, gnm_get_option_group ());
 
-#if defined(G_OS_WIN32)
-	/* we have already translated to utf8, do not do it again.
-	 * http://bugzilla.gnome.org/show_bug.cgi?id=361321 */
-	g_option_context_set_delocalize   (ocontext, FALSE);
-#endif
-
 	g_option_context_add_group (ocontext, gtk_get_option_group (TRUE));
 	g_option_context_parse (ocontext, &argc, &argv, &error);
 
-- 
